python/ray/_private/ray_client_microbenchmark.py

```diff
@@ -69,14 +68,26 @@ def actor_concurrent():
 
 
 def main():
-    system_config = {"put_small_object_in_memory_store": True}
-    with ray_setup_and_teardown(
-            logging_level=logging.WARNING, _system_config=system_config):
-        for name, obj in inspect.getmembers(sys.modules[__name__]):
-            if not name.startswith("benchmark_"):
-                continue
-            with ray_start_client_server() as ray:
-                obj(ray)
+    ray_config = {
+        "_system_config": {
+            "put_small_object_in_memory_store": True
+        },
+        "logging_level": logging.WARNING
+    }
+
+    def ray_connect_handler(job_config=None):
+        import ray as real_ray
```

Comment:
should `import ray` just work? I feel disable_client_hook will take care of everything.

Reply:
Well, if you don't have the import in here and only the disable_client_hook, the behavior is actually unexpected (and fails).

So it's important to have *some form of import* in here, but I felt that `real_ray` was more readable and would prevent scoping problems

Reply:
Ah, you misunderstood me.
I mean we don't necessarily rename it to `real_ray` since it doesn't quite make sense here.

If you don't call  with disable_client_hook, it's actually not real_ray, right?

Reply:
OK i tried a small fix here!